app/Domain/Integrations/Events/IntegrationCreatedPost.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations\Events;
+
+use Illuminate\Foundation\Events\Dispatchable;
+use Ramsey\Uuid\UuidInterface;
+
+final class IntegrationCreatedPost
```

Comment:
What does `Post` mean in this context?

Reply:
"After". I will take suggestions on this name. :-)

Reply:
Renamed it to `IntegrationCreatedAfter`

Reply:
That is still a name that doesn't make much sense. You want to make clear that the contacts are also created, so something like `IntegrationCreatedWithContacts` can be fired from the repository.

Reply:
Oké, I renamed it